docs/blog/2020-02-18-gazette-2019-recap/index.md

```diff
@@ -0,0 +1,178 @@
+
+### 2.14.0: Better Split Chunks Config
+
+#### [2019-08-26](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.13.83...gatsby@2.14.0)
+
+Inspired by [Next.js](https://nextjs.org/), [Ward Peeters](https://twitter.com/wardpeet) added better split chunk config to Gatsby's base webpack config, making your site chunks more efficient.
+
+### 2.15.0: Node Materialisation
+
+#### [2019-08-30](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.14.7...gatsby@2.15.0)
+
+Another big one! [Mikhail Novikov](https://twitter.com/freikesnet) rewrote much of Gatsby's Node model to support arbitrary data stores in the future. This was done to enable faster data stores in the future and larger Gatsby sites.
+
+### 2.16.0: Structured Logging
+
+#### [2019-10-14](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.15.37...gatsby@2.16.0)
+
+[Michał Piechowiak](https://twitter.com/mipiechowiak) and [Sid Chatterjee](https://twitter.com/chatsidhartha) released their work on the Structured logging effort in this release. Most of this was a ground-up rewrite of the way Gatsby prints logs and activities, enabling alternative UIs for Gatsby including a Gatsby Desktop app in the future.
+
+### 2.17.0: App Data
+
+#### [2019-10-21](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.16.5...gatsby@2.17.0)
+
+[Alex Fenton](https://twitter.com/afeno90) (contributor extraordinaire) moved the webpack compilation hash in Gatsby over to a separate file. This was a step forward for Anthony's effort in 2.9.0, making site uploads incremental and reducing the number of changed files per build.
+
+### 2.18.0: Schema Rebuilding
+
+#### [2019-11-19](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.17.17...gatsby@2.18.0)
+
+[Vladimir Razuvaev](https://twitter.com/vrazuvaev) (who recently joined the Gatsby team) killed it in his first month (😱) and added support for automatic Schema rebuilding in the Gatsby CLI. This previously required users to restart the gatsby CLI when changing the schema of their gatsby sites.
+
+There’s a _lot_ more that’s not included here including error message upgrades, several performance improvements, React prerelease tests, an in progress TS migration and more.
+
+## 2020
+
+We started off this year on a solid note with the release of [Gatsby Builds](https://www.gatsbyjs.org/blog/2020-01-27-announcing-gatsby-builds-and-reports/), the _fastest_ way to build your Gatsby sites in the cloud. In OSS land, we've done plenty already as well.
+
+### Internationalizing Gatsby Docs
+
+Nat resumed her effort of helping internationalize the Gatsby documentation. We want Gatsby to be the most accessible way to build for the web. If you're interested in contributing translations in a language of your choice, go check out the translation contribution guidelines!
```

Comment:
Seems a little inconsistent to not use twitter links here, but ok

Reply:
Oops, I totally missed this one. Adding!